js/fill-form.js

```diff
@@ -2,26 +2,26 @@ const typeEstateElement = document.querySelector('#type');
 const pricePerNightElement = document.querySelector('#price');
 const timeInSelectElement = document.querySelector('#timein');
 const timeOutSelectElement = document.querySelector('#timeout');
-const bungalowMinPrice = 0;
-const flatMinPrice = 1000;
-const houseMinPrice = 5000;
-const palaceMinPrice = 10000;
+const BUNGALOW_MIN_PRICE= 0;
+const FLAT_MIN_PRICE= 1000;
+const HOUSE_MIN_PRICE = 5000;
+const PALACE_MIN_PRICE = 10000;
 
 
 typeEstateElement.addEventListener('click', function(e) {
   e.preventDefault();
   if (typeEstateElement.value == 'bungalow') {
```

Comment:
везде и всегда строгое сравнение
```suggestion
  if (typeEstateElement.value === 'bungalow') {
```